scripts/zones/Mine_Shaft_2716/mobs/Moblin_Clergyman.lua

```diff
@@ -0,0 +1,60 @@
+-----------------------------------
+-- Area: Mine Shaft 2716
+-- Quest: Return to the Depths
+-- NM: Moblin Clergyman (x3)
+-----------------------------------
+local entity = {}
+
+local function getBoss(mob)
+    local entityQuery = mob:getZone():queryEntitiesByName('Twilotak')
+    local results = {}
+    for _, r in pairs(entityQuery) do
+        if r:isMob() then
+            table.insert(results, r)
+        end
+    end
+
+    return results[mob:getBattlefield():getArea()]
+end
+
+entity.onMobInitialize = function(mob)
+end
+
+entity.onMobSpawn = function(mob)
+    mob:setMobMod(xi.mobMod.NO_MOVE, 1)
+    mob:setMobMod(xi.mobMod.NO_AGGRO, 1)
+    mob:setMobMod(xi.mobMod.MAGIC_COOL, 90)
+
+    mob:addImmunity(xi.immunity.SLEEP)
```

Comment:
Dont use SLEEP immunity. Use LIGHT_SLEEP and DARK_SLEEP

Reason: regular SLEEP immunity is planned to be retired

Reply:
Fixed, thanks!